test/users.spec.js

```diff
@@ -282,3 +290,74 @@ describe('/users/sessions [POST]', () => {
     });
   });
 });
+
+describe('/users [GET]', () => {
+  let token = {};
+  beforeAll(async () => {
+    const userLogin = {
+      email: 'user@wolox.com.ar',
+      password: 'hola1234'
+    };
+    userLogin.password = await hashPassword('hola1234');
+    await createUser(userLogin);
+    userLogin.password = 'hola1234';
+    token = await (await postUser('/users/sessions', userLogin)).body.token;
+  });
+
+  describe('It should require authorization', () => {
+    let response = {};
+    beforeAll(async () => {
+      await createUser();
+      await createUser();
+      await createUser();
+      response = await request
+        .get('/users?offset=1&limit=2')
+        .send()
+        .set('Accept', 'application/json');
+    });
+
+    it('Receive status code 401.', () => {
+      expect(response.statusCode).toBe(401);
+    });
+  });
+
+  describe('With authorization token', () => {
```

Comment:
To test pagination with page and limit sended or not, you should define:
describe('Successful without pagination params ....
describe('Successful with pagination params ....

Also, check the response params and the keys in response